### PR TITLE
odim 72:Adding the changes related redis stream publish in aggregation , task and lib message bus

### PR DESCRIPTION
**Pull request by jeevan-kamkar**

### CODE REVIEW

File: lib-messagebus/datacommunicator/header.go [hunk 1]
```diff
@@ -58,30 +59,35 @@ type MsgProcess func(d interface{})
...
 
 // Communicator defines the Broker platform Middleware selection and corresponding
 // communication object would be created to send / receive the messages. Broker
 // type would be stored as part of Connection Object "Packet".
 // TODO: We would be looking into Kafka Synchronous communication API for providing
 // support for Sync Communication Model in MessageBus
-func Communicator(bt int, messageQueueConfigPath, pipe string) (MQBus, error) {
+func Communicator(bt string, messageQueueConfigPath, pipe string) (MQBus, error) {
 
 	// Defining pointer for KAFKA Connection Objects Based on
```
Comment (irappa-pattar) [inline lib-messagebus/datacommunicator/header.go:72]: Add RedisStream info also in comment section

File: lib-messagebus/datacommunicator/header.go [hunk 2]
```diff
@@ -28,7 +28,8 @@ import (
 // BrokerType defines the underline MQ platform to be selected for the
 // messages. KAFKA is the platform supported as part of odimra phase 1.
```
Comment (irappa-pattar) [inline lib-messagebus/datacommunicator/header.go]: update this comment section

**Review by irappa-pattar — APPROVED** [review]
minor comments given, please address

File: lib-messagebus/datacommunicator/header.go [hunk 3]
```diff
@@ -26,9 +26,10 @@ import (
 )
 
 // BrokerType defines the underline MQ platform to be selected for the
-// messages. KAFKA is the platform supported as part of odimra phase 1.
+// messages. KAFKA and RedisStremas platforms are supported.
 const (
-	KAFKA = iota // KAFKA as Messaging Platform, Please use this ID
+	KAFKA        = "Kafka"        // KAFKA as Messaging Platform, Please use this ID
+	REDISSTREAMS = "RedisStremas" // REDISSTREAMS as Messaging Platform
```
Comment (amar-shalgar) [inline lib-messagebus/datacommunicator/header.go]: Please correct the spelling

**Review by amar-shalgar — CHANGES_REQUESTED** [review]
Please address changes

File: lib-messagebus/datacommunicator/header.go (hunk 3 above)
Comment (jeevan-kamkar) [reply lib-messagebus/datacommunicator/header.go]: updated

**Review by amar-shalgar — APPROVED** [review]
lgtm

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**